common/djangoapps/track/middleware.py

```diff
@@ -70,7 +83,12 @@ def process_request(self, request):
 
             views.server_track(request, request.META['PATH_INFO'], event)
         except:
-            pass
+            ## Why do we have the overly broad except?
```

Comment:
Early on we decided that an instrumentation problem would not cause a user facing error.

I agree with your strategy here, though. I like the idea of tracking the request if there is an error.